services/card_service.rb

```diff
@@ -0,0 +1,60 @@
+class CardService
+  extend InputHelper
+  extend OutputHelper
+  extend Uploader
+
+  class << self
+    def create_card(accounts, current_account)
```

Comment:
you pass `current_account` and `accounts` to almost all methods, don't you think it would be just easier to just create an instance of CardService with account(s) info ?

Reply:
Done